2021/yearbook.html

```diff
@@ -140,6 +140,11 @@ <h2>2021 Software Development Engineer Bootcamp</h2>
           <figcaption>Nolitha</figcaption>
         </figure>
       </a>
+      <a href="https://github.com/BoipeloM-coder/BoipeloM-github.io" target="_blank" >
```

Comment:
Your website must be something like https://boipelom-coder.github.io pointing to your Github Pages and your Github Repo